worlds/tunic/er_scripts.py

```diff
@@ -300,6 +314,10 @@ def pair_portals(world: "TunicWorld") -> Dict[Portal, Portal]:
         portal_pairs[portal1] = portal2
         two_plus.remove(portal1)
 
+        # add zig skip exit to the dependent regions
+        dependent_regions[("Zig Skip Exit",)] = ["Zig Skip Exit", "Rooted Ziggurat Lower Front",
+                                                 "Rooted Ziggurat Lower Back", "Rooted Ziggurat Portal Room Entrance"]
```

Comment:
It's a bit hard to keep track, but does this result in world bleed issues? Since you set dependent_regions to something static by direct assignment further up:

`dependent_regions = dependent_regions_restricted`

and here you add to it

Reply:
I think even if that is the case it might work out in this case because you always set it to the same thing but it looks suspicious

Reply:
Good catch, it should probably be a copy instead here